sprocs/assessment_instances_insert.sql

```diff
@@ -1,9 +1,10 @@
-DROP FUNCTION IF EXISTS assessment_instances_insert(bigint,bigint,bigint,enum_mode,integer,timestamp with time zone);
+DROP FUNCTION IF EXISTS assessment_instances_insert(bigint,bigint,boolean,bigint,enum_mode,integer,timestamp with time zone);
```

Comment:
Hi Matt, I am currently fixing those in my local versions and Could you give me the permission to modify this PR so that I could mark this one resolved?

Reply:
@zhining4 I added you to PL with `write` access (@mwest1066 hope I didn't overstep)

Reply:
@tdy. Thanks for that, I could edit now.